go/ai/generate.go

```diff
@@ -25,6 +25,7 @@ import (
 
 	"github.com/firebase/genkit/go/core"
 	"github.com/firebase/genkit/go/core/logger"
+	"github.com/firebase/genkit/go/internal"
```

Comment:
internal is kind of a weird package name, usually it is internal/something. Perhaps internal/schema?
Maybe ok though?

Reply:
How about internal/base, like cmd/compile/internal/base?

Reply:
will do in a followup.

Reply:
That seems fine.